src/coreclr/jit/codegencommon.cpp

```diff
@@ -2315,7 +2315,7 @@ void CodeGen::genReportEH()
 
         CORINFO_EH_CLAUSE_FLAGS flags = ToCORINFO_EH_CLAUSE_FLAGS(HBtab->ebdHandlerType);
 
-        if (isNativeAOT && (XTnum > 0))
+        if (XTnum > 0)
         {
             // For NativeAOT, CORINFO_EH_CLAUSE_SAMETRY flag means that the current clause covers same
```

Comment:
```suggestion
            // CORINFO_EH_CLAUSE_SAMETRY flag means that the current clause covers same
```
Delete mentions of native AOT since this flag is not going to be native AOT specific anymore?

Reply:
Also, here: https://github.com/dotnet/runtime/blob/bc70fec98fe39e63e194e7ffa30e9d1f44b4f254/src/coreclr/inc/corinfo.h#L910

Reply:
I was going to update the comment in the actual PR for the EH as that's where the usage of this flag will start to happen, but I can do it here if you prefer.

Reply:
I think it is fine to do it here.